CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [1.18](https://github.com/brave/brave-ios/releases/tag/v1.18)
+
+ - Implemented enhancments to new tab page and favourites overlay. ([#2578](https://github.com/brave/brave-ios/issues/2578))
+ - Implemented enhancments to favorites to use fallback monogram letters instead of low-resoltuion icons. ([#2579](https://github.com/brave/brave-ios/issues/2579))
```

Comment:
* `enhancments ` should be `enhancements`
* `low-resoltuion` should be `low-resolution`